README.md

```diff
@@ -71,6 +97,24 @@ The file `rust-mode-tests.el` contains tests that can be run via
 You can use `run_rust_emacs_tests.sh` to run them in batch mode, if
 you set the environment variable EMACS to a program that runs emacs.
 
+# LSP
+
+## eglot
+
+```elisp
+(require 'eglot)
```

Comment:
same here. even if the setup is very simple, it can end up being out of sync with upstream.